mods/ra/rules/defaults.yaml

```diff
@@ -989,6 +1023,8 @@
 	Immobile:
 		OccupiesSpace: true
 	HitShape:
+	EditorTilesetFilter:
+		Categories: Mine
```

Comment:
I don't think it's worth giving these their own category.  How about putting these in System too?